amlip_cpp/src/cpp/types/AmlipGenericTopicDataType.hpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
Not sure if this should be updated for autogenerated files. Anyway, apply the same criteria for all of them.